content/blog/2023-04-18-the-value-of-composability-in-ethereum-classic/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "The Value of Composability in Ethereum Classic"
+date: 2023-04-18
+author: Donald McIntyre
+contributors: ["DonaldMcIntyre"]
+tags: ["education"]
+linkImage: ./etc-composability.png
+---
+
+---
+**You can listen to or watch this video here:**
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Alxz8pDfMhg" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+---
+
+## What Is Composability?
+
+In traditional systems, when you try to book a flight, a hotel, a car rental, and pay everything at once with a credit card with the condition that the dates and prices you reserved are exactly what you selected, the process of closing and paying that composed transaction is very difficult technologically because all the services are hosted in many isolated datacenters of the different providers. 
+
+Consequently, it is very likely that the transaction will be partitioned and executed sequentially with a high risk that any of the service terms may have changed or with no availability by the time the transactions are processed in each system.
+
+If all these services were on a blockchain as Ethereum Classic (ETC) they would be hosted in the same system in the form of smart contracts and the transaction would be executed instantaneously in a single block.
+
+This ability of a system to host all the services in the same place as modular components that may interact together in many ways and combinations is called composability.
+
+![A single database.](./etc-composability.png)
+
+From a developer perspective, composability is also extremely valuable because all the ecosystem of smart contracts inside ETC becomes a collection of services that may be reused and leveraged, thus reducing development time and increasing innovation.
+
+All the components and services inside ETC are like lego blocks that both users and developers may put together as they please. This has enabled a very unique experience in this type of technology.
+
+This level of composability is very likely an unintended benefit not anticipated by Satoshi Nakamoto when he invented Bitcoin, although he did envision that it would have smart contracts eventually, an idea that proved very difficult come true due to that system’s design.
```

Comment:
Line 33: Missing word
```suggestion
This level of composability is very likely an unintended benefit not anticipated by Satoshi Nakamoto when he invented Bitcoin, although he did envision that it would have smart contracts eventually, an idea that proved very difficult to come true due to that system’s design.
```